### PR TITLE
wscl-issues: Add more current practices

### PR DESCRIPTION
**Pull request by yitzchak**
Also fix a small typo in the test case of the clrhash-type-error issue.

### CODE REVIEW

File: wscl-issues/proposed/find-package-type-error [hunk 1]
```diff
@@ -29,7 +30,8 @@ Test Cases:
     (declare (optimize (safety 3)))
     (find-package object))
 
-  (one 234) => ERROR: The value 234 is not of type ...
+    (one 234) => [signals TYPE-ERROR with expected type
+                   (OR CHARACTER PACKAGE STRING SYMBOL)]
```
Comment (scymtym) [inline wscl-issues/proposed/find-package-type-error:34]: Indentation looks wrong here.

**Review by scymtym — CHANGES_REQUESTED** [review]
One very minor issue in a commit comment.

File: wscl-issues/proposed/find-package-type-error (hunk 1 above)
Comment (yitzchak) [reply wscl-issues/proposed/find-package-type-error]: Fixed.

**Review by scymtym — APPROVED** [review]

**Comment by scymtym** [comment]
Looks good to me, thanks.

**Pull request merged**